pages/treeherder.py

```diff
@@ -78,6 +84,11 @@ def filter_by(self, term):
         el.send_keys(Keys.RETURN)
         self.wait.until(lambda s: self.result_sets)
 
+    def filter_unclassified_jobs(self):
+        self.find_element(*self._unclassified_failure_filter_locator).click()
+        self.wait.until(lambda s: len(self.result_sets) == 10)
+        return self
```

Comment:
Hey @davehunt! Am I understanding correctly that there is no need to `return self`?
If so should we remove `return self` from `get_next_(ten/twenty/fifty)_results` function? Please check above rows 66, 71, 76.